### PR TITLE
[ADD] New module account_invoice_analytic_quick_view

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
Add a link to the project into the account invoice if all the invoice line have the same account

### CODE REVIEW

File: account_invoice_project/README.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================
+Account Invoice Project
+=======================
+
+This module provides a new computed fields into account.invoice model
```
Comment (sbidoul) [inline account_invoice_project/README.rst]: Please provide a clear description. eg what field? how is it computed? why is it useful?

File: account_invoice_project/README.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    :alt: License: AGPL-3
+
+=======================
+Account Invoice Project
+=======================
+
+This module provides a new stored and computed fields on account.invoice.
+This fields contains the project related to the invoice lines if the lines
+have the same project. In case where the lines are linked to different
+projects then the new computed fields will be "False"
```
Comment (sbidoul) [inline account_invoice_project/README.rst]: s/"False"/empty.

File: account_invoice_project/README.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================
+Account Invoice Project
```
Comment (sbidoul) [inline account_invoice_project/README.rst]: Project -> Analytic Account

File: account_invoice_project/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "name": "Account Invoice Project",
+    "version": "8.0.1.0.0",
+    "author": "ACSONE SA/NV",
+    "category": "Accounting & Finance",
+    "website": "http://acsone.eu",
+    "depends": [
+        "account",
+        "project",
+    ],
+    "summary": """Account Invoice Project""",
```
Comment (sbidoul) [inline account_invoice_project/__openerp__.py]: Project -> Analytic Account

File: account_invoice_project/models/account_invoice.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    def onchange_invoice_line(self):
+        self.account_analytic_id = self._get_account_analytic_id(
+            self.invoice_line)
+
+    # new field
+    account_analytic_id = fields.Many2one(
+        compute='_account_analytic_id',
+        comodel_name="account.analytic.account",
+        string="Analytic account",
+        store=True)
```
Comment (sbidoul) [inline account_invoice_project/models/account_invoice.py]: readonly?

File: account_invoice_project/views/account_invoice_view.xml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+                <xpath
+                    expr="//page[@string='Other Info']/group/group/field[@name='name']"
+                    position="before">
+                    <field name="account_analytic_id" />
+                </xpath>
+                <field name="invoice_line" position="attributes">
+                    <attribute name="context">
+                        {'default_account_analytic_id':
+                        account_analytic_id}
+                    </attribute>
```
Comment (sbidoul) [inline account_invoice_project/views/account_invoice_view.xml]: I would remove this one

File: account_invoice_project/views/account_invoice_view.xml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        <!-- customize invoice filters -->
+        <record model="ir.ui.view" id="view_account_invoice_filter">
+            <field name="name">account.invoice.select
+                (account_invoice_project)
+            </field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.view_account_invoice_filter" />
+            <field name="arch" type="xml">
+                <field name="partner_id" position="after">
+                    <field name="account_analytic_id" string="Project" />
```
Comment (sbidoul) [inline account_invoice_project/views/account_invoice_view.xml]: question is it possible to filter in invoice_line.account_analytic_id?

File: account_invoice_project/models/account_invoice.py (hunk 5 above)
Comment (JonathanNEMRY) [inline account_invoice_project/models/account_invoice.py]: It is a computed field so the readonly attribute is not required

File: account_invoice_project/views/account_invoice_view.xml (hunk 7 above)
Comment (JonathanNEMRY) [inline account_invoice_project/views/account_invoice_view.xml]: Seems that field must be a field name. It is possible to do it with force_domain (something like that) or maybe into a domain but I am afraid that's not possible for this case

**Pull request merged**